summarization/text_entity_detection.py

```diff
@@ -0,0 +1,45 @@
+import base64
+import json
+import os
+
+import requests
+
+from summarization.bad_request_error import BadRequestError
+
+
+class TextEntityRetriever:
+
+    const_api_endpoint \
```

Comment:
Constant variables are generally written in CAPITAL_CASE. Also, "const_" won't be required.

Reply:
Oh i see. Thanks for the suggestion!

Reply:
+1

Reply:
updated accordingly!